esp/scm/ws_workunits.ecm

```diff
@@ -1434,7 +1434,8 @@ ESPresponse [exceptions_inline] WUQueryFilesResponse
 
 ESPrequest WUQueryDetailsRequest
 {
-    string QueryId;
+    [depr_ver("1.61")] string QueryId;
+    [min_ver("1.61")] string QueryIdOrAlias;
```

Comment:
@wangkx No, that kind of confuses interface version with implementation version.  I think keeping it one parameter but adding support for aliases in the background is ok, at least in this case.